lib/versioned/printers/printer.js

```diff
@@ -42,6 +42,21 @@ TestPrinter.prototype.maybePrint = function maybePrint() {
   }
 }
 
+TestPrinter.prototype.printVersionedMatrix = function printVersionedMatrix() {
+  console.log('Versions executed\n')
```

Comment:
We should add this file as an exception to not warn for `console.logs` since its purpose is printing. Not needed for this PR (unless) you want. Just calling out.

Reply:
I'll do it in the other PR I'm about to open